tests/pfcwd/test_pfcwd_function.py

```diff
@@ -119,9 +119,15 @@ def update_alpha(dut, port, profile, value):
         """
         logger.info("Updating dynamic threshold for {} to {}".format(profile, value))
         asic = dut.get_port_asic_instance(port)
+
+        if PfcCmd.isBufferInApplDb(asic):
+            db = "0"
+        else:
+            db = "4"
+
         asic.run_redis_cmd(
             argv = [
-                "redis-cli", "-n", "4", "HSET", profile, "dynamic_th", value
+                "redis-cli", "-n", db, "HSET", profile, "dynamic_th", value
```

Comment:
@ppikh Can you check the format of `profile` as well? IMO they differ between `CONFIG_DB` and `APPL_DB`.
- In `CONFIG_DB`, it should be like `BUFFER_PROFILE|<profile_name>`
- In `APPL_DB`, it should be like `BUFFER_PROFILE_TABLE:<profile_name.`

Reply:
Hi,
I checked by doing "show techsupport" and comparing configs in dump folder. They looks ok, i've not seen broken config.

Reply:
I checked the code.
`profile` is passed from `self.profile` which has been initialized from `PfcCmd.get_mmu_params(self.dut, port)`.
In `PfcCmd.get_mmu_params` there is the logic of fetching the buffer table entry from `APPL_DB` or `CONFIG_DB` according to whether the dynamic buffer calculation is enabled or not.
So `profile` contains the correct buffer profile and the logic should be fine.